addons/mail/wizard/mail_template_preview_views.xml

```diff
@@ -6,39 +6,36 @@
-                            <span class="col-md-5 col-lg-4 col-sm-12 ps-0">Force a language: </span>
-                            <div class="col-md-7 col-lg-6 col-sm-12 ps-0">
-                                <field name="lang" placeholder="Select a language" class="w-100"/>
-                            </div>
+                        <div class="alert alert-danger" role="alert" attrs="{'invisible' : [('error_msg', '=', False)]}">
+                            <field name="error_msg" />
                         </div>
-                    </div>
-                    <group>
-                        <field name="subject"/>
-                        <field name="email_from" attrs="{'invisible':[('email_from','=', False)]}"/>
-                        <field name="partner_ids" widget="many2many_tags" attrs="{'invisible':[('partner_ids', '=', [])]}"/>
-                        <field name="email_to" attrs="{'invisible':[('email_to','=', False)]}"/>
-                        <field name="email_cc" attrs="{'invisible':[('email_cc','=', False)]}"/>
-                        <field name="reply_to" attrs="{'invisible':[('reply_to','=', False)]}"/>
-                        <field name="scheduled_date" attrs="{'invisible':[('scheduled_date','=', False)]}"/>
-                    </group>
-                    <field name="body_html" widget="html" nolabel="1" options='{"safe": True}'/>
-                    <field name="attachment_ids" widget="many2many_binary"/>
+                        <field name="no_record" invisible="1"/>
+
+                        <group>
+                            <label for="resource_ref" string="Test Record: "/>
+                            <field name="resource_ref" readonly="False" nolabel="1"
+                                   options="{'hide_model': True, 'no_create': True, 'no_open': True}"
+                                   attrs="{'invisible': [('no_record', '=', True)]}"/>
+                            <b attrs="{'invisible': [('no_record', '=', False)]}" class="text-warning">No record for this model</b>
+
+                            <field name="lang" string="Force a Language:" placeholder="Select a language"/>
+
+                            <field name="subject"/>
+                            <field name="email_from" attrs="{'invisible':[('email_from','=', False)]}"/>
+                            <field name="partner_ids" widget="many2many_tags" attrs="{'invisible':[('partner_ids', '=', [])]}"/>
+                            <field name="email_to" attrs="{'invisible':[('email_to','=', False)]}"/>
+                            <field name="email_cc" attrs="{'invisible':[('email_cc','=', False)]}"/>
+                            <field name="reply_to" attrs="{'invisible':[('reply_to','=', False)]}"/>
+                            <field name="scheduled_date" attrs="{'invisible':[('scheduled_date','=', False)]}"/>
+
+                            <field name="body_html" widget="html" colspan="2" nolabel="1" options='{"safe": True}'/>
```

Comment:
For consistency, would use double quotes around the attribute "options" and single quotes inside.

Reply:
Yes indeed, thanks.